modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/controls/CreationMenu.tsx

```diff
@@ -14,6 +14,7 @@ import {triggerAction} from '../../utils/actionItems/triggerAction';
 
 export interface ICreationActionItem {
 	data?: {
+		action?: string;
```

Comment:
The whole object of this interface in an action item. It's not clear  to me what `action` attribute of an action should represent. Should this be `id`?